tests/common.go

```diff
@@ -703,6 +707,35 @@ func ValidateContextForPureVolumesSDK(ctx *scheduler.Context, errChan ...*chan e
 				})
 			}
 		})
+
+		//check mount option attribute in storage class
+		Step("validating mount options", func() {
+			if ctx.SkipMountOptionValidation {
+				return
+			}
+			ctx.SkipVolumeValidation = false
+			ValidateContext(ctx)
+			ctx.SkipVolumeValidation = true
+			vols, err := Inst().S.GetVolumes(ctx)
+			processError(err, errChan...)
+			for _, vol := range vols {
+				pvcObj, err := k8sCore.GetPersistentVolumeClaim(vol.Name, vol.Namespace)
+				sc, err := k8sCore.GetStorageClassForPVC(pvcObj)
+				if err != nil {
+					log.Errorf("Error Occured while getting storage class for pvc %v", err)
```

Comment:
if you do not want to continue in case of error here, please use log.FailOnError

Reply:
Done.